src/main.py

```diff
@@ -0,0 +1,60 @@
+import asyncio
+import base64
+import json
+import os.path
+import websockets
+
+with open(os.path.join("resources", "script.pdf"), mode="rb") as f:
```

Comment:
The global stuff in this file is just to have something that works. I want to replace it with something nicer when we read start reading the opus from a file.